dashboard/app.R

```diff
@@ -214,10 +157,64 @@ ui <- fluidPage(padding=0,
 
 
 server <- function(input, output, session) {
+  # Connect to AWS s3bucket
+  Sys.setenv("AWS_DEFAULT_REGION" = "us-east-2")
+  s3bucket = tryCatch(
+    {
+      get_bucket(bucket = 'forecast-eval')
+    },
+    error = function(e) {
+      e
+      return(NULL)
+    }
+  )
   
-  ##############
-  # CREATE PLOTS
-  ##############
+  # Get and prepare data
+  getData <- function(filename){
+    if(!is.null(s3bucket)) {
+      tryCatch(
+        {
+          s3readRDS(object = filename, bucket = s3bucket)
```

Comment:
I think this may read this for every new session.  The score cards are pretty small, but it could introduce a load time problem eventually.

Reply:
The methods of reloading the app from within the app that I've seen do not actually restart the app, so I think this code has to be within the server to get re-run, unless there is another way of doing it that I haven't seen yet.